README.md

```diff
@@ -8,29 +8,30 @@ To quickly learn how to run cleanlab on your own data, first check out the [quic
 
```

Comment:
add this notebook to `ignore_folders` in:

https://github.com/cleanlab/examples/blob/master/run_all_notebooks.py

This will prevent auto-execution of this notebook. We auto-execute all other notebooks in this repo whenever we push new stable release of cleanlab package to ensure they run well.